Proposed clarification changes to terminology (#126)
tfpauly/draft-pauly-adaptive-dns-privacy: draft-pauly-oblivious-doh.md

This document defines the following terms:

Oblivious Resolution Server:
: A server that acts as either a Proxy or Target.

Oblivious Proxy:
: An ODoH server that proxies encrypted DNS queries and responses between a client and an ODoH Target. 

Oblivious Target:

> : An ODoH server that receives and decrypts encrypted client DNS queries from a Proxy, and returns encrypted DNS responses via that same  Proxy. A Target may be co-located with a resolver or part of the resolver itself, but neither is a requirement.

Reviewer: ```suggestion
: An ODoH server that receives and decrypts encrypted client DNS queries from a Proxy, and returns encrypted DNS responses via that same Proxy. In order to provide DNS responses, the target can be a DNS resolver, be co-located with a resolver, or forward to a resolver.
```
